### PR TITLE
Update README.md with instructions to live_grep todos

### PR DESCRIPTION
**Pull request by champignoom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,20 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
 <video alt="find_aof_project_tasks" src="https://user-images.githubusercontent.com/81827001/158401841-9ca3a311-bac1-4733-9a6e-6125003d8a38.mov"></video>
 </details>
 
+Until the new GTD infrastructure arrives, you could collect all your todo items with a regex, for example:
+
+```lua
+-- Add the following function to your `~/.config/nvim/init.lua`:
+function get_todos(dir, states)
+     require('telescope.builtin').live_grep{cwd=dir}
```
Comment (max397574) [inline README.md:74]:
```suggestion
     require('telescope.builtin').live_grep{ cwd = dir }
```
format please

File: README.md [hunk 2]
```diff
@@ -66,6 +66,20 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
 <video alt="find_aof_project_tasks" src="https://user-images.githubusercontent.com/81827001/158401841-9ca3a311-bac1-4733-9a6e-6125003d8a38.mov"></video>
 </details>
 
+Until the new GTD infrastructure arrives, you could collect all your todo items with a regex, for example:
+
+```lua
+-- Add the following function to your `~/.config/nvim/init.lua`:
+function get_todos(dir, states)
```
Comment (max397574) [inline README.md:73]:
```suggestion
local function get_todos(dir, states)
```

File: README.md [hunk 3]
```diff
@@ -66,6 +66,20 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
...
+     require('telescope.builtin').live_grep{cwd=dir}
+     vim.fn.feedkeys('^ *([*]+|[-]+) +[(]' .. states .. '[)]')
+end
+
+-- You can now use `:lua get_todos('~/notes', '[^x_]')` to get all todo items inside directory '~/notes' that are not yet done or cancelled.
```
Comment (max397574) [inline README.md:78]:
```suggestion
```
mapping is enough
if anything more then a command
we don't want users to have to call lua functions like this

File: README.md [hunk 4]
```diff
@@ -66,6 +66,20 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
...
+-- Or, to bind it to a key, like ctrl_t:
```
Comment (max397574) [inline README.md:79]:
```suggestion
-- This can be bound to a key
```

**Review by max397574 — CHANGES_REQUESTED** [review]
1. will this work ootb or require e.g. ripgrep? and a certain matcher?
2. would be really cool if you could automatically detect the current workspace (either just in the keymapping or in the function itself)
if you don't know how tell me and I'll search a snippet for you (you should be able to find plenty in this repo)

**Comment by champignoom** [comment]
done and done

File: README.md [hunk 5]
```diff
@@ -66,6 +66,26 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
...
 
+Until the new GTD infrastructure arrives, you could collect all your todo items with a regex, for example:
+
+```lua
+-- Make sure you have ripgrep installed.
+-- Add the following function to your `~/.config/nvim/init.lua`:
+do
+    local _, neorg = pcall(require, "neorg.core")
+    local dirman = neorg.modules.get_module("core.dirman")
+    local current_workspace = dirman.get_current_workspace()
```
Comment (max397574) [inline README.md:77]: needs to be inside the function
otherwise it won't work if you switch workspaces

**Review by max397574 — APPROVED** [review]

**Pull request merged**